### PR TITLE
Geotools/geo address search label update

### PR DESCRIPTION
**Pull request by mdastous-bentley**

### CODE REVIEW

File: packages/geo-tools/public/locales/en/GeoTools.json
Comment (calebmshafer) [inline packages/geo-tools/public/locales/en/GeoTools.json:3]: Let's hold this until I fix the current publishing issues after @aruniverse's fix.  I'll have a PR up in a couple minutes.

**Review by calebmshafer — CHANGES_REQUESTED** [review]

File: packages/geo-tools/public/locales/en/GeoTools.json
Comment (calebmshafer) [reply packages/geo-tools/public/locales/en/GeoTools.json:3]: Once #87 is merged this should be good to go.

**Review by calebmshafer — APPROVED** [review]

**Review by aruniverse — APPROVED** [review]

**Pull request merged**
